internal/pkg/scaffold/cr.go

```diff
@@ -38,7 +38,7 @@ type CR struct {
 func (s *CR) GetInput() (input.Input, error) {
 	if s.Path == "" {
 		fileName := fmt.Sprintf("%s_%s_%s_cr.yaml",
-			strings.ToLower(s.Resource.Group),
+			strings.ToLower(s.Resource.GoImportGroup),
```

Comment:
Nit: Doesn't really matter, but we don't have to use `strings.ToLower()` here, since we've already done that for `GoImportGroup`:

https://github.com/operator-framework/operator-sdk/blob/4db8cd7e21a9a9a672cc0996f96bd9dfe1d29d66/internal/pkg/scaffold/resource.go#L133-L134